lib/build.js

```diff
@@ -333,6 +333,12 @@ export default {
         throw new BuildError('Invalid build file.', 'No executable command specified.');
       }
 
+      return Promise.resolve().then(() => {
+        if (target.preBuild && _.isFunction(target.preBuild)) {
```

Comment:
I want to move away from using lodash, this condition could be reduced to `typeof target.preBuild === 'function'`.